esx_service/vmci/vmci_client.c

```diff
@@ -291,6 +291,8 @@ vsock_get_reply(be_sock_id *s, be_request *r, be_answer* a)
    ret = recv(s->sock_id, a->buf, b, 0);
    if (ret == -1 || ret != b) {
       free(a->buf);
+      a->buf = NULL;
+      fprintf(stderr, "Failed to receive message data: received %d expected %d\n", ret, b);
```

Comment:
Do these errors go to /var/log/messages or /var/log/syslog.log? We should update CI to dump contents of these files so that we can troubleshoot failed test cases.